src/classes/STG_InstallScript.cls

```diff
@@ -369,6 +377,21 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
 
+    /*******************************************************************************************************
+    * @description Creates the finalized opp stage to state record based on what value is set as the 
+    * "Opportunity Stage for When Fully Paid" 
+    */
+    private void createOppStageToStateRecordsIfNecessary() {
```

Comment:
We've never tested creating new CMT records as part of the install script process. This will need to be tested in a beta package to ensure that it works as expected. There's always the possibility that this method needs to be handled as a Queueable executed from the PostInstallScript.

Reply:
@force2b - Would you prefer we start out doing this as a Queuable, rather than the current approach, or are you ok with trying it first.  Agree on how we will have to test it, unfortunately.

Reply:
You can try it first, though it might be because MixedDML rules apply with calls to the FeatureManagement namespace I suspect that it will need to be handled asynchronously. So might be easier to just start out as async.

Reply:
Hi @force2b We're utilizing the "Metadata.Operations.enqueueDeployment()" which is already async. Since it's already async, I don't think the MixedDML rules should be an issue. I'm curious to see if this will actually work as expected in the post install script process though. This is a reference link: 

https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_metadata_retrieve_deploy.htm